native/acter/src/api/profile.rs

```diff
@@ -179,14 +172,7 @@ impl RoomProfile {
             .client
             .get_room(&self.room_id)
             .context("Room not found")?;
-        let format = match thumb_size {
-            Some(thumb_size) => MediaFormat::Thumbnail(MediaThumbnailSize {
-                method: get_content_thumbnail::v3::Method::Scale,
-                width: thumb_size.width(),
-                height: thumb_size.height(),
-            }),
-            None => MediaFormat::File,
-        };
+        let format = into_media_format(thumb_size);
```

Comment:
As per the API change suggested above 
```suggestion
        let format = ThumbnailSize::parse_into_media_format(thumb_size);
```